.github/workflows/delete-package.yml

```diff
@@ -14,8 +14,9 @@ on:
           required: true
           default: 'ballerina-lang'
         package:
-          description: 'Specific package'
+          description: 'Specific package(groupId.artifactId)'
           required: true
+          default: 'org.ballerinalang.jballerina-tools'
```

Comment:
Lets remove the default. There can be scenarios, where this can be mistakenly dropped.

Reply:
fixed